.github/workflows/deployGHPages.yml

```diff
@@ -13,10 +13,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js
       uses: actions/setup-node@v2
       with:
-        node-version: 16.14.2
+        node-version: 20.x
```

Comment:
I preferred fixed version, I recently have an issue on a build with the new `18.18`. Maybe `20.7.x` is good enough.